draft-bbs-signatures.md

```diff
@@ -604,11 +602,11 @@ Procedure:
 
 8. cv = HASH(PK || Abar || A' || D || C1 || C2 || pm)
 
-9. if c != cv return INVALID
+9. if c != cv, return INVALID
 
-10. if A' == 1 return INVALID
+10. if A' == 1, return INVALID
```

Comment:
Need to check Abar != 1

Reply:
Can you clarify why this check is needed?? If it's to avoid passing 1 in the parings i suggest even removing the A' != 1 check and have a more unified way of performing this check for all elements.

Reply:
Yes that’s why. I agree to the unified method

Reply:
Referenced in #126